models/migrations/v46.go

```diff
@@ -0,0 +1,24 @@
+// Copyright 2017 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package migrations
+
+import (
+	"fmt"
+
+	"code.gitea.io/gitea/models"
+
+	"github.com/go-xorm/xorm"
+)
+
+func addIssueDependencyTables(x *xorm.Engine) (err error) {
+
+	err = x.Sync(new(models.IssueDependency))
```

Comment:
This should be a separate struct so we don't have collisions in future updates

Reply:
You mean, a seperate copy of `models.IssueDependency`?

Reply:
Preferably inside this function yes :)

Reply:
done 😁